site/app/controllers/admin/NotebookBuilderController.php

```diff
@@ -43,12 +43,11 @@ public function builder(string $g_id, string $mode) {
         }
 
         $failure_url = $this->core->buildCourseUrl(['gradeable', $gradeable->getId(), 'update']) . '?nav_tab=1';
-
-        if (!$gradeable->isUsingUploadedConfig()) {
-            $this->core->addErrorMessage("Notebook builder may only edit uploaded configurations for the current course and semester.");
-            return new RedirectResponse($failure_url);
-        }
-
+        // Related to PR #5945
+        // if (!$gradeable->isUsingUploadedConfig()) {
+        //     $this->core->addErrorMessage("Notebook builder may only edit uploaded configurations for the current course and semester.");
+        //     return new RedirectResponse($failure_url);
+        // }
```

Comment:
remove comments

Reply:
Removed this check, moved it further down to line 85